.github/workflows/deb_validator.yaml

```diff
@@ -0,0 +1,68 @@
+name: Verify debian packages
+
+on:
+  push:
```

Comment:
Any situation where we would push w/o PR?

Reply:
It is a nice to have to push the branch and work without a pr sometimes